composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^4.5 || ^5.4",
-        "guzzlehttp/psr7": "^1.3"
+        "guzzlehttp/psr7": "^1.3",
+        "php-http/message": "^1.0"
```

Comment:
It will not work because the discovery will fail. It will ask you to install a message factory. When you installed the message factory it will work. 

I cant, and should not, require php-http/message because the message factory the user wants to use may be in a different package.